### ISSUE TITLE
Is it a good idea to use latest version in all packages?

### ISSUE THREAD

**Issue by shirohana** [nuxt-community/modules#165]
I see bulma was already in `0.6.1` but in `@nuxtjs/bulma` was just `^0.5.1`.

Since you use a helper module, that means you didn't care about which version you
used but it should be the newest one.

I mean, if you want to use the older version, just look around how `@nuxtjs/bulma`
was made and specify the version when adding it, and no needs to send a issue or pr
for upgrading dependencies in `@nuxtjs/modules` anymore.

```
"dependencies": {
  "bulma": "latest"
}
```

https://github.com/nuxt-community/modules/blob/master/packages/bulma/package.json#L12

**Comment by clarkdo** [comment]
I think the modules should not depend these frameworks directly, maybe `peerDependencies` is a choice, we can let users add specific version dependency by themselves.

**Comment by anteriovieira** [comment]
@clarkdo I agree with you, in that case, we would have to implement `peerDependencies` on all modules, right?

**Comment by clarkdo** [comment]
@anteriovieira Yeah, we may change it one by one and keep discussing. 😸

**Comment by anteriovieira** [comment]
Okay, so I'm going to start with a pull request for  `@nuxtjs/bulma` :smiley:
